README.md

```diff
@@ -2,10 +2,11 @@
 
 <div align="left">
 
+[![Documentation](https://readthedocs.org/projects/xrmocap/badge/?version=latest)](https://xrmocap.readthedocs.io/en/latest/?badge=latest)
 [![actions](https://github.com/openxrlab/xrmocap/workflows/build/badge.svg)](https://github.com/openxrlab/xrmocap/actions)
 [![codecov](https://codecov.io/gh/openxrlab/xrmocap/branch/main/graph/badge.svg)](https://codecov.io/gh/openxrlab/xrmocap)
 [![PyPI](https://img.shields.io/pypi/v/xrmocap)](https://pypi.org/project/xrmocap/)
-[![LICENSE](https://img.shields.io/github/license/openxrlab/xrmocap.svg)](https://github.com/openxrlab/xrmocap/blob/main/LICENSE)
+[![Percentage of issues still open](https://isitmaintained.com/badge/open/open-mmlab/xrmocap.svg)](https://github.com/open-mmlab/xrmocap/issues)
```

Comment:
`https://github.com/open-mmlab/xrmocap/issues` is not a valid link.

Reply:
Good catch.

Reply:
And the issue link?
https://github.com/openxrlab/xrmocap/issues

Reply:
Update issue link in https://github.com/openxrlab/xrmocap/pull/12/commits/2c7c8fb9fc5892a5ad8bacfa84f783e68b7ce682.